.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,7 @@
 ## Description
 
-<!--- Describe your changes in detail -->
+## Demo
 
-## Related Issue
+## Review notes
 
-<!--- This project only accepts pull requests related to open issues -->
-<!--- If suggesting a new feature or change, please discuss it in an issue first -->
-<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
-<!--- Please link to the issue here: -->
-
-## How Has This Been Tested?
-
-<!--- Please describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran to -->
-<!--- see how your change affects other areas of the code, etc. -->
-
-## Checklist
```

Comment:
Why remove Checklist?

Reply:
Current checklist doesn't make a lot of sense

> - [ ] Requires other services change
> - [ ] Affects to other services

This project is mostly standalone, if there will be dependencies, we will be able to add it

> - [ ] Requires dependency update

This is just not needed for any reason

> - [ ] Automated tests

No tests right now

> - [ ] Looks good on large screens
> - [ ] Looks good on mobile

Looks good is a bit stretched term, feels like demo section will be enough.